tests/cross/src/main/scala/tests/pc/BaseTypeDefinitionSuite.scala

```diff
@@ -0,0 +1,27 @@
+package tests.pc
+
+import tests.BasePCSuite
+import scala.concurrent.duration.Duration
+
+class BaseTypeDefinitionSuite extends BasePCSuite {
```

Comment:
Doesn't look like we need this as a separate class altogether - it's only utility methods used in one suite.

Reply:
Changed.